backend/estimate-service/src/main/java/org/egov/works/service/WorkflowService.java

```diff
@@ -41,7 +40,7 @@ public class WorkflowService {
     public BusinessService getBusinessService(EstimateRequest estimateRequest) {
         String tenantId = estimateRequest.getEstimate().getTenantId();
         String rootTenantId = tenantId.split("\\.")[0];
```

Comment:
Now, we do not need the concept of root tenantId in the workflow service. In the current state also, it will work even if we have configured workflow at `pb` and passing the tenantId `pb.amritsar`.

Reply:
ok..i'll change this in next PR